code/modules/mob/living/carbon/human/species/abductor.dm

```diff
@@ -36,4 +36,11 @@
 	H.gender = NEUTER
 	H.languages.Cut() //Under no condition should you be able to speak any language
 	H.add_language("Abductor Mindlink") //other than over the abductor's own mindlink
-	return ..()
\ No newline at end of file
+	var/datum/atom_hud/abductor_hud = huds[DATA_HUD_ABDUCTOR]
+	abductor_hud.add_hud_to(H)
+	return ..()
+
+/datum/species/abductor/remove_abilities(mob/living/carbon/human/H)
```

Comment:
this is gross, but about the only thing that's valid right now.

As I spoke to @Aurorablade. Once the species refactor is determined to be fairly stable, I'll probably implement a few more things from her PR so this makes more sense.